templates/firefox-ios.xliff

```diff
@@ -620,13 +465,21 @@
         <note>The link that shows more Pocket trending stories</note>
       </trans-unit>
       <trans-unit id="ActivityStream.Pocket.SectionTitle">
-        <source>Recommended by Pocket</source>
+        <source>Trending on Pocket</source>
         <note>Section title label for Recommended by Pocket section</note>
       </trans-unit>
       <trans-unit id="ActivityStream.Pocket.Trending">
         <source>Trending</source>
         <note>The description of a Pocket Story</note>
       </trans-unit>
+      <trans-unit id="ActivityStream.TopSites.RowCount">
+        <source>%d rows</source>
+        <note>label showing how many rows of topsites are shown. %d represents a number</note>
+      </trans-unit>
+      <trans-unit id="ActivityStream.TopSites.RowSettingFooter">
+        <source>Set Rows</source>
+        <note>The link that shows more Pocket trending stories</note>
```

Comment:
Just want to make extra sure the localization comment is accurate here? I'm unsure if they correspond.
Also, is "Set" here used as a verb?